TypeCobol.Analysis/Cfg/ControlFlowGraphBuilder.MultiBranchContext.cs

```diff
@@ -35,28 +31,49 @@ internal class MultiBranchContext
             /// The Index in the SuccessorEdge of the RootBlock if any, -1 otherwise.
             /// </summary>
             internal int RootBlockSuccessorIndex;
-
+            /// <summary>
+            /// Terminals block associated to Multi branch Context if any.
+            /// Terminals blocks are block that have as successor NextFlowBlock or
+            /// a branching to the beginning of a loop instruction block.
+            /// </summary>
+            public IList<BasicBlock<Node, D>> Terminals { get; internal set; }
             /// <summary>
             /// The instruction associated to this context.
             /// </summary>
-            internal Node Instruction;
-
+            public Node Instruction { get; internal set; }
             /// <summary>
             /// List to hold When, WhenOther, WhenSearch and AtEnd nodes
             /// encountered in Evaluate or Search statements.
             /// </summary>
             internal List<Node> ConditionNodes;
+            /// <summary>
+            /// Block of the next flow
+            /// </summary>
+            public BasicBlock<Node, D> NextFlowBlock { get; internal set; }
+            /// <summary>
+            /// Any Sub context if any, null otherwise.
+            /// For instance in COBOL EVALUATE Context has all WHEN and WHENOTHER as sub contexts.
+            /// </summary>
+            public IList<IMultiBranchContext<Node, D>> SubContexts { get; internal set; }
+
+            private BasicBlock<Node, D> RootBlockForEnd;
+
+            BasicBlock<Node, D> IMultiBranchContext<Node, D>.OriginBlock => this.OriginBlock;
+
+            BasicBlock<Node, D> IMultiBranchContext<Node, D>.RootBlock => this.RootBlock;
+
+            IList<BasicBlock<Node, D>> IMultiBranchContext<Node, D>.Branches => this.Branches;
+
+            IList<int> IMultiBranchContext<Node, D>.BranchIndices => this.BranchIndices;
```

Comment:
MultiBranchContext is an internal class of ControlFlowGraphBuilder<D>, so public members are accesed via a public interface IMultiBranchContext<Node, D>,  explicit interface are use to pubish public attribute that are declared with an internal type or an embedded type.

Reply:
But why mix both implementation styles then ? Some of the properties are publicly implemented and those 4 are treated differently. To summarize my view on this:
- do not create the interface type, it is not relevant
- make `MultiBranchContext` public
- check and change visbility for members: make members that should be accesible from outside public and everything else should be internal

Reply:
Ok as seen together, this is a trick to avoid exposing the `BasicBlockForNode` class, so we can keep it like that except for the last one which can easily be converted into public implementation.

Reply:
done in 0067e69